gordo_components/cli/cli.py

```diff
@@ -19,6 +19,12 @@
 
 from gordo_components.builder.build_model import ModelBuilder
 from gordo_components import serializer
+from gordo_components.builder.mlflow_utils import (
+    get_workspace_kwargs,
+    get_spauth_kwargs,
+    mlflow_context,
+    log_metadata,
```

Comment:
Possible to import `mlflow_utils` and use the functions from there? For me, that's easier to follow the code